src/components/MigrationNotice.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <NoticeBar>
+    {{ $t('migration-notice.intro') }}
+    <i18n tag="span" path="migration-notice.more">
+      <template #read-more>
+        <a
+          class="text-dark-blue hover:text-dark-blue underline"
+          href="https://wordpress.org/news/2021/05/welcome-to-openverse/"
+          target="_blank"
+          >{{ $t('migration-notice.read') }}</a
+        >
```

Comment:
Why does prettier/eslint format components this way, with the closing tag split across two lines? I've never seen this happen in React, it looks very strange to read this way.

Reply:
I think it's because of how Vue compiler treats newlines as spaces. This combined with [Whitespace-sensitive formatting](https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting) prevents links from having an extra space at either end.

Reply:
Oh I see, that makes sense.